examples/todoapp/bin/server/controllers/translations_controller.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:shelf/shelf.dart';
+
+import '../repositories/translations_repository.dart';
+import '../utils/api_controller.dart';
+
+class TranslationsController extends ApiController {
```

Comment:
@DDavidPrime don't we have a translations controller in the rx_bloc_cli ?

Reply:
Is this rx_bloc_cli generated code? https://github.com/Prime-Holding/rx_bloc/blob/7a44e708a85fad6148622f3b69fcc2091872149c/examples/todoapp/bin/server/controllers/translations_controller.dart
It looks like the initial scaffold creation.